Civi/Api4/Generic/AbstractQueryAction.php

```diff
@@ -77,14 +77,15 @@ abstract class AbstractQueryAction extends AbstractAction {
    * @param string $fieldName
    * @param string $op
    * @param mixed $value
+   * @param bool $isExpression
    * @return $this
    * @throws \API_Exception
    */
-  public function addWhere(string $fieldName, string $op, $value = NULL) {
+  public function addWhere(string $fieldName, string $op, $value = NULL, bool $isExpression = FALSE) {
     if (!in_array($op, CoreUtil::getOperators())) {
       throw new \API_Exception('Unsupported operator');
     }
-    $this->where[] = [$fieldName, $op, $value];
+    $this->where[] = [$fieldName, $op, $value, $isExpression];
     return $this;
   }
```

Comment:
This new param actually doesn't apply to every entity. Only those with a sql table. So I think this function should be left as-is here, and the new version copied over to `DAOGetAction`

Reply:
Agreed, that makes more sense.